File: src/main/java/io/zeitwert/ddd/validation/model/impl/AggregatePartValidationImpl.java
package io.zeitwert.ddd.validation.model.impl;

import io.zeitwert.ddd.validation.model.AggregatePartValidation;
import io.zeitwert.ddd.validation.model.enums.CodeValidationLevel;
import lombok.Builder;
import lombok.Data;

@Data
@Builder
public class AggregatePartValidationImpl implements AggregatePartValidation {

	private Integer seqNr;
	private CodeValidationLevel validationLevel;
	private String validation;

}

File: fm-dddrive/src/main/java/dddrive/ddd/validation/model/impl/AggregatePartValidationImpl.kt
package dddrive.ddd.validation.model.impl

import dddrive.ddd.validation.model.AggregatePartValidation
import dddrive.ddd.validation.model.enums.CodeValidationLevel

class AggregatePartValidationImpl(
	override val seqNr: Int,
	override val validationLevel: CodeValidationLevel,
	override val message: String,
	override val path: String? = null,
) : AggregatePartValidation
